src/radio/index.js

```diff
@@ -138,19 +138,22 @@ class Radio extends EventEmitter {
     }
 
     if (!this.order.some(u => u.equals(user))) return false;
-
-    this.skips.add(user.id);
     this.checkSkips();
+    this.skips.add(user.id);
     return true;
   }
 
   checkSkips() {
-    if (!this.current) return false;
+    if (!this.current) return -1;
```

Comment:
why this change?

Reply:
Oh this was done because I was returning a number for voteskip to check on skip because, at the time, the skip was being done there. So I needed to have it return a number and it would use that to check.

Reply:
Also I suppose I can just go back now